examples/all.yaml

```diff
@@ -245,3 +245,12 @@ fio_git_username: ""
 fio_git_token: ""
 fio_git_branch: ""
 fio_cleanup: true
+
+#cron-job vars
+enable_cronjob: false  # Set true to enable the cronjob
+cj_schedule: "*/1 * * * *"       
+cj_concurrencyPolicy: "Allow"  
+cj_startingDeadlineSeconds: 200   
+cj_suspend: false                 
+cj_successfulJobsHistoryLimit: 4
+cj_failedJobsHistoryLimit: 2
```

Comment:
can we not use camel case to maintain consistency with other variable names?

Reply:
variable names updated